test/SlowTests/Issues/RavenDB-18413.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Operations.ConnectionStrings;
+using Raven.Client.Documents.Operations.ETL;
+using Raven.Client.Documents.Queries.Timings;
+using Raven.Server;
+using Raven.Server.Documents;
+using Raven.Server.Documents.TimeSeries;
+using Raven.Server.ServerWide.Context;
+using Raven.Tests.Core.Utils.Entities;
+using Raven.Server.Config;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_18413 : RavenTestBase
+    {
+        public RavenDB_18413(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task ToQueryableTimings()
```

Comment:
void

Reply:
Huh? We _never_ want `async void` methods

Reply:
Just `void`
```suggestion
        public void ToQueryableTimings()
```

Reply:
Done